sdk/synapse/Azure.Analytics.Synapse.Artifacts/README.md

```diff
@@ -80,17 +80,17 @@ NotebookResource notebookResource = operation.WaitForCompletionAsync().Configure
 `GetNoteBook` retrieves a notebook.
 
 ```C# Snippet:RetrieveNotebook
-NotebookResource notebook = notebookClient.GetNotebook("MyNotebook");
+NotebookResource notebook = client.GetNotebook("MyNotebook");
 ```
 
 ### List notebooks
 `GetNotebooksByWorkspace` enumerates the notebooks in the Synapse workspace.
 
 ```C# Snippet:ListNotebooks
-Pageable<NotebookResource> notebooks = notebookClient.GetNotebooksByWorkspace();
-foreach (NotebookResource notebook in notebooks)
+Pageable<NotebookResource> notebooks = client.GetNotebooksByWorkspace();
+foreach (NotebookResource book in notebooks)
```

Comment:
nit: I think the previous name is more descriptive.

Reply:
Agreed, but there is already a notebook in scope. Suggestion on better name?